tests/integration/tests/test_util/snap.py

```diff
@@ -76,6 +82,11 @@ def get_most_stable_channels(
     for channel, major, minor, risk in arch_flavor_channels:
         version_key = (int(major), int(minor))
 
+        if min_release is not None:
+            _min_release = major_minor(min_release)
+            if version_key < _min_release:
```

Comment:
nit: `_min_release` might be `None` which will cause `TypeError: '<' not supported between instances of 'tuple' and 'NoneType'`
```suggestion
            if _min_release is not None and version_key < _min_release:
```

Reply:
Good catch!